src/main/php/PDepend/TextUI/Command.php

```diff
@@ -344,7 +348,7 @@ protected function assignArguments(): void
         }
 
         // Check for suffix option
-        if (isset($this->options['--suffix'])) {
+        if (isset($this->options['--suffix']) && is_string($this->options['--suffix'])) {
             // Get file extensions
             $extensions = explode(',', $this->options['--suffix']);
```

Comment:
```suggestion
        $suffix = $this->options['--suffix'] ?? null;

        if (is_string($suffix)) {
            // Get file extensions
            $extensions = explode(',', $suffix);
```
Maybe we can do this to repeat `'--suffix'` a bit less (and the same for other options)